aws/modules/core/route53.tf

```diff
@@ -18,6 +18,14 @@ resource "aws_route53_record" "zone_delegation" {
   records = [ "${aws_route53_zone.core.name_servers}" ]
 }
 
+resource "aws_route53_record" "soa" {
+  zone_id = "${aws_route53_zone.core.zone_id}"
+  name = "${var.vpc_name}.${var.dns_domain}"
+  type = "SOA"
+  ttl = "900"
+  records = [ "${aws_route53_zone.core.name_servers.0}. awsdns-hostmaster.amazon.com. 1 7200 900 1209600 ${var.soa_negative_cache_ttl}" ]
```

Comment:
while we're here, do we want to change the email address to something we own? This might make it easier for TLS cert providers to verify certs, for example